dll/shellext/zipfldr/CZipEnumerator.hpp

```diff
@@ -84,10 +84,18 @@ struct CZipEnumerator
         err = unzGetCurrentFileInfo64(uf, &info, NULL, 0, NULL, 0, NULL, 0);
         if (err == UNZ_OK)
         {
-            PSTR buf = name.GetBuffer(info.size_filename);
-            err = unzGetCurrentFileInfo64(uf, NULL, buf, name.GetAllocLength(), NULL, 0, NULL, 0);
-            name.ReleaseBuffer(info.size_filename);
-            name.Replace('\\', '/');
+            CStringA nameA;
```

Comment:
Why do we stick with A here?

Reply:
`unzGetCurrentFileInfo64` below doesn't support UTF-16 (W).